VMBackup/main/handle.py

```diff
@@ -322,7 +327,11 @@ def daemon():
             canTakeCrashConsistentSnapshot = can_take_crash_consistent_snapshot(para_parser)
             temp_g_fsfreeze_on = True
             freeze_snap_shotter = FreezeSnapshotter(backup_logger, hutil, freezer, temp_g_fsfreeze_on, para_parser, canTakeCrashConsistentSnapshot)
-
+            if (hutil.ExtErrorCode == ExtensionErrorCodeHelper.ExtensionErrorCodeEnum.FailedInvalidDataDiskLunList):
+                temp_result = CommonVariables.FailedInvalidDataDiskLunList
+                temp_status = 'error'
+                error_msg = "Input mismatch CRP did not send the LUN No's of disks to be included"
```

Comment:
this is not a mismatch but an invalid input case, right?

Invalid input. IsAnyDiskExcluded is marked as true but input LUN list is empty. It is not allowed if VM has Direct Drives or if VM has Write Accelerated disks or if VM is a TVM/CVM.